### ISSUE TITLE
pac admin backup

### ISSUE THREAD

**Issue by robcaretta-thrivent** [MicrosoftDocs/power-platform#1755]
Please expand the documentation on pac admin backup. What level of access is needed to start this?  I have a service account added to an environment as a System Administrator and it's not able to execute that specific command, however it is able to push new version of the solution to the environment via GitHub action workflows.

_Error: The service principal with id 'xxxxxxxxxxxxxxxx' for application xxxxxxxxxxxxxxxx does not have permission to access the path 'https://xxxxxxxxxxxxx/providers/Microsoft.BusinessAppPlatform/scopes/admin/environments?`$expand=permissions&api-version=2020-08-01' in tenant xxxxxxxxxxxxxxxxxxx._

**Comment by JimDaly** [comment]
@robcaretta-thrivent 

I'm assuming that you are looking for information to be added here: [pac admin backup](https://learn.microsoft.com/en-us/power-platform/developer/cli/reference/admin#pac-admin-backup)

Assigning to @kkanakas 

Please note that these issues are for feedback related to documentation only. We don't handle support requests and don't have resources to troubleshoot issues. If you need a resolution, please contact tech support.

**Comment by robcaretta-thrivent** [comment]
Correct, looking for expanded documentation on that page regarding the required permission. It's not clear to me if System Administrator for an environment would be enough, or if that set of commands are reserved for Power Platform Tenant Admins.

**Comment by JimDaly** [comment]
The description for all the items in the group of **pac admin** commands is "_Work with your Power Platform Admin Account_"

You mention "_I have a service account added to an environment as a System Administrator and it's not able to execute that specific command,..._".

Are you able to use any of the other commands in this group with that account?

The proposal for doc changes is:

1. Confirm whether the user must only be a specified PP admin account. 
2. Expand the description or add additional notes to clarify this.

**Comment by robcaretta-thrivent** [comment]
That's unfortunate, but I'm fine with that answer. Our goal is build a reusable set of scripts that teams can use to manage their own repos and environment. We will have to rethink that type of request then, it will not work as a part of their automation.

I think some alignment is needed here though. If you review the parameters it allows for an app ID and secret. So I would presume by that it allows for using an "application user" within the environment. That doesn't line up with the pac admin documentation. 
https://github.com/microsoft/powerplatform-actions/blob/main/backup-environment/action.yml

I haven't tried any of the other pac admin commands via GitHub workflows, but I have been able to use that application user to pull from an unmanaged solution in one environment and push it to another environment as managed via these actions. 
microsoft/powerplatform-actions/import-solution@v0
microsoft/powerplatform-actions/export-solution@v0

**Comment by JimDaly** [comment]
Please don't take this as an answer. I'm not SME in this area. Just suggesting what we might do to the docs to address this.

**Comment by sanjeev40084** [comment]
@robcaretta-thrivent i had the same issue, here is what worked for me: https://github.com/microsoft/powerplatform-vscode/issues/393 (note: i won't say this is the best solution since the solution didn't work at first but tried the same thing (running the backup command next day) and it worked). hope that helps.
